app/controllers/items_controller.rb

```diff
@@ -4,9 +4,13 @@ def new
     type = params[:type]
     druid = params[:druid]
     targets = params[:targets]
+    raise 'missing required params' unless type && druid && targets
```

Comment:
Yeah, I just thought it gives you a framework provided way to handle required/optional params to a controller.